pkg/windows/windows.go

```diff
@@ -408,6 +411,10 @@ func (vm *windows) ConfigureKubeProxy(nodeName, hostSubnet string) error {
 	return nil
 }
 
+func (vm *windows) StopKubelet() error {
+	return vm.ensureServiceNotRunning(&service{name: kubeletServiceName})
```

Comment:
@mansikulkarni96 is it safe to do this or do I need to stop all services by calling `vm.ensureRequiredServicesStopped()`?

Reply:
I think you will need to call vm.ensureRequiredServicesStopped() before doing vm.ensureServiceNotRunning()  otherwise we will run into an error from WMCB : trying to stop a service that other services are dependent on.

Reply:
`vm.ensureRequiredServicesStopped()` ensures that the kubelet is not running, so why would we need to call `vm.ensureServiceNotRunning(&service{name: kubeletServiceName})` in addition?

Reply:
that's right we don't need to do it in addition, edited my comment a moment ago.

Reply:
Ah you said instead. Sorry I must have misread.